xmodule/tests/test_word_cloud.py

```diff
@@ -42,7 +45,11 @@ def test_xml_import_export_cycle(self):
 
         olx_element = etree.fromstring(original_xml)
         runtime.id_generator = Mock()
-        block = WordCloudBlock.parse_xml(olx_element, runtime, None)
+
+        def_id = runtime.id_generator.create_definition(olx_element.tag, olx_element.get('url_name'))
+        keys = ScopeIds(None, olx_element.tag, def_id, runtime.id_generator.create_usage(def_id))
+        block = WordCloudBlock.parse_xml(olx_element, runtime, keys)
```

Comment:
As per discussion in pair programming session we are creating scope ids here which shouldn't happen.

Reply:
Aha, here's why you needed to make this change! Passing in `None` for ScopeIds was a legacy behavior: https://github.com/openedx/edx-platform/blob/6d150acf81b07c72e85bc86e2171cffc3a7818a7/xmodule/xml_block.py#L309-L312

This change looks good 👍🏻